src/test/java/create_task_use_case/CreateTaskDsInputDataTest.java

```diff
@@ -0,0 +1,24 @@
+package create_task_use_case;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Task Creation Ds Input Data Test Cases.
+ * @author Fardin Faruk
+ */
+public class CreateTaskDsInputDataTest {
+
+    @Test
+    public void testGetTitle(){
+        CreateTaskDsInputData outputData = new CreateTaskDsInputData("Exam", 0);
```

Comment:
why is the DsInputData variable names outputData?

Reply:
done